WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -36,7 +36,7 @@ def index
           total: users_array.size,
           rows: selected_rows.map do |user|
             {
-              wca_id: user.wca_id ? view_context.link_to(user.wca_id, person_path(user.wca_id)) : "",
+              wca_id: user.wca_id ? view_context.link_to(user.wca_id, person_path(user.wca_id)) : (user.unconfirmed_wca_id ? view_context.link_to(user.unconfirmed_wca_id, edit_user_path(user.id, anchor: "wca_id"), class: "pending-id") : ""),
```

Comment:
For the record: I am firmly opposed to nested ternaries. I think it's always better to pull this sort of construct out into a separate variable or function or something.

Reply:
Indeed. I tried to make an <%if in there but it didn't work. A variable would be better.